### ISSUE TITLE
PS 4 tested &. thx for 1.16 support

### ISSUE THREAD

**Issue by TomTje** [Pugmatt/BedrockConnect#63]
First of all, super nice that you managed to support version 1.16.  On our switch it works again :) On the PS4 I also tried it but unfortunately without success.  I have the feeling that Minecraft doesn't even start because it is stuck at the Mojang screen.  if it continues then Minecraft can not connect to the MS account.  But there are also no servers available for the PS4 like Hive or others.  that's a real shame.

**Comment by Nagol12344** [comment]
ok cool

**Comment by Pugmatt** [comment]
Yeah, the DNS method on PS4 isn't possible for the reason you said. Although I'm pretty sure people have successfully connected to the serverlist through LAN, so I'm unsure why it isn't connecting to your MS account. If you were to get the MS account working, you could potentially get on the serverlist by hosting a local BedrockConnect instance:

- On PC, Mac, or Linux, download this JAR: https://github.com/Pugmatt/BedrockConnect/releases
- Make sure you have Java 8 or higher installed.
- In command prompt or terminal, run the following command:
```java -jar BedrockConnect-1.0-SNAPSHOT.jar nodb=true```

At this point the serverlist server should be running. Make sure this is also running on the same network as the Xbox. When in Minecraft, go to the "Friends" tab, and the server should be showing under LAN servers and available to join.

**Comment by Pugmatt** [comment]
Here's a pretty good video I found of someone getting it work on PS4 using Android: https://www.youtube.com/watch?v=b8cRB88U6wY

**Comment by giggidy45** [comment]
Pugmatt, that's the method I was using before 1.16 for the PS4 as well. But since it also uses your DNS BedrockConnect server it doesn't work atm either sadly

**Comment by Pugmatt** [comment]
That method uses LAN to connect to the server, the DNS trick shouldn't be involved.

**Comment by tsheeley** [comment]
I've got it working on my PS4 Pro using [PocketMine](https://pmmp.readthedocs.io/en/rtfd/installation/installing-manually.html) on Win10 Pro... then I'm using `/transferserver 104.238.130.180` to move to the BC public server.

It works great... it's even able to save additional servers to the list!

The only issue is that CubeCraft crashes the game on my PS4 whenever I try to enter a game-hub area.

It's currently 2:30am... so I've only tried the Survival and Skyblock sections, but I'm guessing the others will crash it as well. Hive: Deathrun and Lifeboat: Skyblock both worked without problems though

Any thoughts on CubeCraft crashing?
